misc/extension_api_validation/4.0-stable_4.1-stable.expected

```diff
@@ -349,3 +349,10 @@ Validate extension JSON: Error: Hash changed for 'classes/EditorUndoRedoManager/
 Validate extension JSON: Error: Hash changed for 'classes/UndoRedo/methods/create_action', from 0AEC1BFC to E87757EB. This means that the function has changed and no compatibility function was provided.
 
 Added a optional parameters with default values. No adjustments should be necessary.
+
+
+GH-84792
```

Comment:
This should be in the 4.1 file, this is for changes between 4.0 and 4.1

Then it'll need to be moved to the 4.2 one when released